configs/textdet/psenet/psenet_resnet50_fpnf_600e_ctw1500.py

```diff
@@ -1,19 +1,16 @@
 _base_ = [
-    'psenet_r50_fpnf.py',
-    '../../_base_/default_runtime.py',
-    '../../_base_/schedules/schedule_adam_step_600e.py',
+    '_base_psenet_resnet50_fpnf.py',
     '../../_base_/det_datasets/ctw1500.py',
+    '../../_base_/textdet_default_runtime.py',
+    '../../_base_/schedules/schedule_adam_step_600e.py',
 ]
 
-# dataset settings
-train_list = {{_base_.train_list}}
-test_list = {{_base_.test_list}}
 file_client_args = dict(backend='disk')
-default_hooks = dict(
-    checkpoint=dict(type='CheckpointHook', interval=100),
-    logger=dict(type='LoggerHook', interval=20))
+# dataset settings
+ctw_det_train = _base_.ctw_det_train
+ctw_det_test = _base_.ctw_det_test
 
-model = {{_base_.model_poly}}
+model = _base_.model_poly
```

Comment:
![image](https://user-images.githubusercontent.com/22607038/185739911-92a3ec80-89d1-4eed-8e3e-e03c7de768d7.png)
![image](https://user-images.githubusercontent.com/22607038/185740196-f89475a6-9c23-4514-ac99-006c74241ed7.png)

In order to be consistent with maskrcnn, we can rename `model_poly` as `model` and pull icdar's pipelines up to the base, then modify `model` and `test_pipeline` in this config. Attached are the only differences between ctw and ic15 configs.